dark/blast/alignments.py

```diff
@@ -147,7 +147,7 @@ def getSubjectSequence(self, title):
         else:
             readClass = DNARead
 
-        return readClass(seq.description, seq.seq)
+        return readClass(seq.description, str(seq.seq))
```

Comment:
I wonder why you need to be explicit here? Isn't it supposed to be a string already?